include/dslash_helper.cuh

```diff
@@ -1,12 +1,16 @@
 #pragma once
 
+#include <color_spinor_field.h>
+#include <gauge_field.h>
 #include <register_traits.h>
 #include <index_helper.cuh>
 #include <tune_quda.h>
+#include <dslash_quda.h>
 
 namespace quda {
 
-  void setPackComms(const int *);
+  void setPackComms(const int *); // original packing kernel
+  void setPackComms2(const int *); // rewriten packing kernel
```

Comment:
Did this get an updated name?

Reply:
There are two `setPackComms` functions.  One is defined in the legacy packer and one in the new packer.  Only one is compiled at once (originally the second variant was used for the new packer).